Source/Diagnostics/BackTransformedDiagnostic.cpp

```diff
@@ -577,7 +577,9 @@ BackTransformedDiagnostic (Real zmin_lab, Real zmax_lab, Real v_window_lab,
       m_particle_slice_width_lab_(particle_slice_width_lab)
 {
 
-
+#ifdef WARPX_DIM_RZ
+    amrex::Abort("BackTransformed diagnostics is currently not supported with RZ");
+#endif
```

Comment:
Do we need the `amrex::Abort` both here and in `MultiDiagnostics.cpp` or can we have just a single one?

Reply:
This is the old BTD :) 
and the MultiDiagnostics.cpp is the new BTD